react/components/utils/index.js

```diff
@@ -7,3 +7,5 @@ export const childrenOf = (...types) => {
 
   return PropTypes.oneOfType([fieldType, PropTypes.arrayOf(fieldType)])
 }
+
+export const isMobileDevice = device => device.toLowerCase() === 'phone'
```

Comment:
Could this be inside the `useDevice`? I find strange the need of using a helper function combined with a hook that supposed to return this information. 🤔

Reply:
Agreed.